### DISCUSSION TITLE
[练打字背单词] 国人项目Qwerty Learner

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#205] [1️⃣ 新鲜事]
- https://qwerty.kaiyi.cool/
- https://github.com/Kaiyiwing/qwerty-learner

第七届开源年会开源百宝箱上看到的项目：

<img width="1322" alt="image" src="https://user-images.githubusercontent.com/32434520/198867045-4846675d-cb6c-43c7-b8bc-fd42b14e091b.png">

**Comment by zhicheng-ning** [comment]
有很多库函数中的词汇还是挺不错的

![image](https://user-images.githubusercontent.com/39022409/198873169-70669d96-ec96-4e2c-b9ad-adcdfa4382f9.png)

**Reply by will-ww** [reply]
So cool~
